data/core/tips.json

```
@@ -288,5 +288,75 @@
     "type": "snippet",
     "category": "tip",
     "text": [ "Don't be afraid to make the game easier on yourself.  Worlds can be made to spawn with more resources." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You're not a fashion model.  Wear whatever it takes to keep you warm and safe, even if it doesn't fit." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "The horde may be numerous, but they are finite.  You can eventually depopulate an area." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "City centers may have more loot, but they also have more powerful zombies." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "A car is more than four wheels and an engine.  You can live out of one if you need to!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "If you're laid up with a broken leg, studying books is a great way to pass the time." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Looking for bionics?  Dissecting more powerful zombies might help you." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "If you've taken everything that isn't nailed down, pull up the nails and knock down the walls!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Having a friend is good, but not everyone left alive wants to be your friend." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Rubber clothes can resist acid." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "If you don't plan to drink the water, odds are good you don't need to clean it either." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "An empty can is still metal.  An empty bottle can be refilled." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Sentry guns are pretty dumb.  Doesn't matter how close you get if they can't see you." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "If it's stupid and it works, it's not stupid." ]
```

Doesn't it sound better with 'but':
```suggestion
    "text": [ "If it's stupid but it works, it's not stupid." ]
```